src/main/scala/com/tapad/docker/ComposeFile.scala

```diff
@@ -247,6 +255,19 @@ trait ComposeFile extends SettingsHelper with ComposeCustomTagHelpers with Print
     }
   }
 
+  def getStaticPortMappings(ports: Seq[String]): java.util.ArrayList[String] = {
```

Comment:
We should handle the case where a user has multiple internal ports that will conflict when exposed externally. The first port should be statically mapped but the remainder that would cause a conflict should by dynamically assigned with a warning message that it could not define a static port X for service Y because port X was already in use.